### PR TITLE
Create README.md for organization overview.

### PR DESCRIPTION
**Pull request by thompsonmj**

### CODE REVIEW

**Comment by hlapp** [comment]
As I said, we shouldn't have divergent definitions of Imageomics in different places, so the one here (which I don't think has been discussed or approved by the leadership team?) needs to be changed.

**Comment by hlapp** [comment]
> As I said, we shouldn't have divergent definitions of Imageomics in different places, so the one here (which I don't think has been discussed or approved by the leadership team?) needs to be changed.

Fixed.

**Comment by hlapp** [comment]
* [ ] @balhoff could you please check the list for Phenoscape and add repositories you think would also specifically fall into the Imageomics context? (Phenex perhaps?)
* [ ] @johnbradley could you add the BGNN repos specifically involved in the Minnow Traits project & manuscript?
* [ ] @thompsonmj can you coordinate with @smrolfe and @muratmaga which repos from their GitHub presence should be linked here? (Perhaps SlicerMorph? ml-morph?)
* [ ] @susan1637 and/or @xbahadirx, could you add (or list in a comment) repos to add here from the TUBRI Github organization?

File: profile/README.md
Comment (balhoff) [inline profile/README.md:30]:
```suggestion
    * [Phenex](https://github.com/phenoscape/Phenex)
```

**Review by balhoff — CHANGES_REQUESTED** [review]
Added Phenex link as suggested by @hlapp.

**Comment by thompsonmj** [comment]
Added products from Murat, Sara, and Arthur.

**Comment by hlapp** [comment]
@susan1637 @xbahadirx @uyedaj last call. We'll wrap this up Thursday at the latest.

**Comment by hlapp** [comment]
@balhoff I don't know whether you previously submitted a formal review requesting changes, but somehow it doesn't recognize the resolved comment as having resolved the requested change. When you get a chance, can you submit an approval?

**Comment by hlapp** [comment]
> @balhoff I don't know whether you previously submitted a formal review requesting changes, but somehow it doesn't recognize the resolved comment as having resolved the requested change. When you get a chance, can you submit an approval?

Scratch that. I just found a way to dismiss your earlier review (even if in actuality I didn't dismiss it 🧐)

**Comment by hlapp** [comment]
@cvstewart @dboghrat @tanyabw once merged, this will appear as part of our [Github organization profile](https://github.com/imageomics). In addition to the version control diff, you can also [see the file in HTML rendering](https://github.com/Imageomics/.github/blob/037a49672b138137b247c9f3046053b332aeb7e7/profile/README.md) (which is how it would appear).

My main ask here is to check whether the content is all OK to go live, or whether (and which) further corrections need to be made. (You can see in the thread here who contributed.)

My hunch is that it is too much information to fit well on the organization profile, and that we'll probably want to move some of this to a different place, such as the website. But we can deal with bite-sizing it more in a subsequent step.

**Review by dboghrat — APPROVED** [review]

**Pull request merged**
